src/pages/workspace/members/ImportedMembersPage.tsx

```diff
@@ -0,0 +1,167 @@
+        {text: translate('common.ignore'), value: CONST.CSV_IMPORT_COLUMNS.IGNORE},
+        {text: translate('common.email'), value: CONST.CSV_IMPORT_COLUMNS.EMAIL, isRequired: true},
+        {text: translate('common.role'), value: CONST.CSV_IMPORT_COLUMNS.ROLE},
+    ];
+
+    const requiredColumns = columnRoles.filter((role) => role.isRequired).map((role) => role);
+
+    const validate = useCallback(() => {
+        const columns = Object.values(spreadsheet?.columns ?? {});
+        let errors: Record<string, string | null> = {};
+
+        if (!requiredColumns.every((requiredColumn) => columns.includes(requiredColumn.value))) {
+            // eslint-disable-next-line rulesdir/prefer-early-return
+            requiredColumns.forEach((requiredColumn) => {
+                if (!columns.includes(requiredColumn.value)) {
+                    errors.required = translate('spreadsheet.fieldNotMapped', requiredColumn.text);
+                }
+            });
+        } else {
+            const duplicate = findDuplicate(columns);
+            if (duplicate) {
+                errors.duplicates = translate('spreadsheet.singleFieldMultipleColumns', duplicate);
+            } else {
+                errors = {};
+            }
+        }
+
+        return errors;
+    }, [requiredColumns, spreadsheet?.columns, translate]);
+
+    const importMembers = useCallback(() => {
+        setIsValidationEnabled(true);
+
+        const errors = validate();
+        if (Object.keys(errors).length > 0) {
+            return;
+        }
+
+        const columns = Object.values(spreadsheet?.columns ?? {});
+        const membersEmailsColumn = columns.findIndex((column) => column === CONST.CSV_IMPORT_COLUMNS.EMAIL);
+        const membersRolesColumn = columns.findIndex((column) => column === CONST.CSV_IMPORT_COLUMNS.ROLE);
+        const membersEmails = spreadsheet?.data[membersEmailsColumn].map((email) => email);
+        const membersRoles = membersRolesColumn !== -1 ? spreadsheet?.data[membersRolesColumn].map((role) => role) : [];
+        const members = membersEmails?.slice(containsHeader ? 1 : 0).map((email, index) => {
+            let role: string = CONST.POLICY.ROLE.USER;
+            if (membersRolesColumn !== -1 && membersRoles?.[containsHeader ? index + 1 : index]) {
+                role = membersRoles?.[containsHeader ? index + 1 : index];
+            }
+
+            return {
+                email,
+                role,
+            };
+        });
+
+        if (members) {
+            setIsImporting(true);
+            importPolicyMembers(policyID, members);
+        }
+    }, [validate, spreadsheet, containsHeader, policyID]);
+
+    const spreadsheetColumns = spreadsheet?.data;
+    if (!spreadsheetColumns) {
+        return;
+    }
+
+    const closeImportPageAndModal = () => {
+        setIsImporting(false);
+        closeImportPage();
+        Navigation.navigate(ROUTES.WORKSPACE_CATEGORIES.getRoute(policyID));
```

Comment:
I think `WORKSPACE_CATEGORIES` is a mistake, right?

Reply:
Yes of course, I missed it when copy pasting haha!